assets/js/options.js

```diff
@@ -7,46 +7,53 @@ $(document).ready(function () {
       $("#main").hide();
     });
 
-    chrome.storage.sync.get(null, function (data) {
+    chrome.storage.local.get(null, function (data) {
       deferred.resolve(data);
     });
+
     return deferred.promise();
   }
 
   $.when(getSettings()).done(function (data) {
     $("#spinner").hide();
     $("#main").show();
-    for (var key in data) {
+
+    for (const key in data) {
+      if (key.startsWith("_")) {
```

Comment:
Which keys start with _ that this applies to?

Reply:
There is an item set (`_syncIgnore`) that allows the storage change listener to be able to ignore changes from the synchronization firing. Basically, an update to `local` will fire an update to `sync`, but that update to `sync` will want to fire an update back to `local`. I used `_syncIgnore` to mark the change as an synchronization so that the listener knows to ignore it.